src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+export default function operate(one, two, operator) {
+  if (one === '0' && two === '0') {
+    return {};
+  }
+  const firstNumber = Big(one);
+  const secondNumber = Big(two);
+
+  const operatorMap = {
+    '+': Big.prototype.plus,
+    '-': Big.prototype.minus,
+    x: Big.prototype.times,
```

Comment:
why this 
x: Big.prototype.times,
and this
'*': Big.prototype.times,
These two seem to perform the same function. Let us be more consistent, since our hash table is one with string as it keys, I suggest with stick to that ie`'x': Big.prototype.times,`.